runtime/src/accounts.rs

```diff
@@ -463,18 +464,19 @@ impl Accounts {
         Ok(account_indices)
     }
 
-    pub fn load_accounts(
+    pub fn load_transaction_batch(
         &self,
         ancestors: &Ancestors,
         txs: &[SanitizedTransaction],
         lock_results: Vec<TransactionCheckResult>,
         hash_queue: &BlockhashQueue,
-        error_counters: &mut ErrorCounters,
+        batch_error_counters: &RwLock<ErrorCounters>,
         rent_collector: &RentCollector,
         feature_set: &FeatureSet,
     ) -> Vec<TransactionLoadResult> {
         txs.iter()
             .zip(lock_results)
+            .par_bridge()
```

Comment:
This is broken, the iteration order will change but the `Vec<TransactionLoadResult>` return value must be in order